From: Felix Fietkau <nbd@nbd.name>
Date: Wed, 4 May 2022 12:03:42 +0200
Subject: [PATCH] PCI: mediatek-gen3: change driver name to mtk-pcie-gen3

This allows it to coexist with the other mtk pcie driver in the same kernel

Signed-off-by: Felix Fietkau <nbd@nbd.name>
---

--- a/drivers/pci/controller/pcie-mediatek-gen3.c
+++ b/drivers/pci/controller/pcie-mediatek-gen3.c
@@ -1031,7 +1031,7 @@ static struct platform_driver mtk_pcie_d
 	.probe = mtk_pcie_probe,
 	.remove = mtk_pcie_remove,
 	.driver = {
-		.name = "mtk-pcie",
+		.name = "mtk-pcie-gen3",
 		.of_match_table = mtk_pcie_of_match,
 		.pm = &mtk_pcie_pm_ops,
 	},
